baseline/pytorch/classify/model.py

```diff
@@ -127,68 +127,154 @@ def predict(self, batch_dict, raw=False, dense=False, **kwargs):
             results.append(outcomes)
         return results
 
-    def get_labels(self):
+    def get_labels(self) -> List[str]:
```

Comment:
If we are going a large overhaul of the baseline models do it make sense to remove the fact that the models hold the list of labels? Like instead of passing in the labels the model could just get the number of labels and the labels are basically just another vocab? it seems like that would move towards the tensor in tensor out model idea

Reply:
seems minor, could be revisited in a future PR.  Would rather not decide in this PR

Reply:
Makes sense